cdap-proto/src/main/java/co/cask/cdap/proto/artifact/AppRequest.java

```diff
@@ -41,19 +41,19 @@ public AppRequest(ArtifactSummary artifact) {
   }
 
   public AppRequest(ArtifactSummary artifact, @Nullable T config) {
-    this(artifact, config, null, null, true);
+    this(artifact, config, null, null, null);
```

Comment:
we want to differentiate between "not-provided" vs "provided" so have to keep this null here.